python/src/aac/plugins/gen_protobuf/gen_protobuf_impl.py

```diff
@@ -362,6 +374,58 @@ def _convert_camel_case_to_snake_case(camel_case_str: str) -> str:
     return snake_case_str
 
 
+def _sanitize_message_to_camel_case(message_with_space: str) -> str:
+    """
+    Remove spaces from messages to convert them to CamelCaseStr.
+
+    Args:
+        message_with_space: the message containing a space to convert
+
+    Returns:
+        CamelCaseStr
+    """
+    message_to_camel_case = ""
+
+    message_to_camel_case = message_with_space.replace(" ", "")
```

Comment:
While I prefer simpler solutions, we can't expect our users to adhere to a particular format or naming convention in their models. We should make this a little more flexible.

It should be fair to assume that any use of whitespace, hyphens, or underscores might serve as breaks in variable names. Let's use those to infer breaks in a name and do some basic formatting that way. So, the example string "some_poor Message-name" should serve as a decent example of what a user could input. I'd suggest splitting the incoming string based on those three criteria, which you can accomplish with a regex split like so:
```python
>>> split_string = re.split(r'\s|-|_|;', "some_poor Message-name")
>>> print(split_string)
['some', 'poor', 'Message', 'name']
```

I'd loop through those split sections then apply the correct caseing for each word.